Tests/govuk_ios/govuk_ios_unit_tests/Mocks/Extensions/MockURLOpening.swift

```diff
@@ -0,0 +1,15 @@
+import UIKit
+
+@testable import govuk_ios
+
+class MockURLOpening: URLOpening {
+    var _stubbedCanOpenUrl: Bool = false
+    func canOpenURL(_ url: URL) -> Bool {
+        return _stubbedCanOpenUrl
+    }
+
+    var _receivedOpenCompletion: Bool = false
+    func open(_ url: URL, options: [UIApplication.OpenExternalURLOptionsKey : Any], completionHandler completion: ((Bool) -> Void)?) {
```

Comment:
Can you please knock these down to new lines?

Reply:
Done